tests/e2e/vsan_stretched_cluster.go

```diff
@@ -916,6 +957,408 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		// TODO: List orphan volumes
 
 		ginkgo.By("Bring up the primary site")
+		if len(fds.hostsDown) > 0 {
+			siteRestore(true)
```

Comment:
Add logic in 'JustAferEach()` to bring up all hosts under `fds.hostsDown` using `powerOnHostParallel()` and the removing partitioning from the hosts under `fds.hostsPartitioned` using `networkFailureParallel()`